sched/scheduler/stateful_scheduler.go

```diff
@@ -145,6 +153,7 @@ func NewStatefulScheduler(
 		runnerFactory: rf,
 		asyncRunner:   async.NewRunner(),
 		addJobCh:      make(chan jobAddedMsg, 1),
+		killJobCh:     make(chan jobKillRequest, 1000), // TODO - what should this value be?
```

Comment:
per @jasonbs10's prev comment - leave at 1; fixed in his PR? should probably double check that his fix would include this new ch and not just addJobCh, or merge the fix